cdap-ui/app/cdap/styles/variables.scss

```diff
@@ -103,3 +103,24 @@ $height-of-footer: 53px;
 
 // EntityListView variables
 $overview-width: 626px;
+
+/*
+  New start. Define colors used throughout CDAP. Use these colors from now on.
+*/
+
+$blue-01: #045599;
+$blue-02: #0076dc;
+$blue-03: #0099ff;
+$blue-04: #58b7f6;
+$blue-05: #7cd2eb;
+$blue-06: #caE7ef;
+$blue-grey01: #454a57;
+$blue-grey02: #4e5568;
+$blue-grey03: #5d6789;
+$blue-grey04: #979fbb;
+$blue-grey05: #bac1d8;
+$blue-grey06: #dce0ea;
+$orange02: #fa8a00;
+$orange03: #ffa727;
+$orange04: #ffcc80;
+$orange05: #ffe0b2;
```

Comment:
can we make this naming consistent? `orange02` or `orange-02`. Also skipping `orange01`?

Reply:
`orange01` was `cdap-orange`. But we should have it consistent. Have added this.